## Fix missing syslog include.
## Originally 04_fix_syslogh_inclusion.dpatch by Nico Golde <nion@debian.org>
--- a/pam_ssh.c
+++ b/pam_ssh.c
@@ -1,4 +1,4 @@
-/*-
+/*
  * Copyright (c) 2006 Wolfgang Rosenauer
  * All rights reserved.
  * 
@@ -62,6 +62,7 @@
 #include <pwd.h>
 #include <signal.h>
 #include <stdio.h>
+#include <syslog.h>
 #include <stdlib.h>
 #include <string.h>
 #include <sysexits.h>
